docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>comment out the gpu line since this will be run on cpu. also is the mem fraction thing needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done in #1290. No it is not needed.